src/util.js

```diff
@@ -0,0 +1,33 @@
+
+const fs = require('fs')
+const getSize = require('get-folder-size')
+
+/**
+  *  Collection of utilities to measure important KPIs
+  */
+class Util {
+  constructor (ipfsDir) {
+    this.ipfsDir = ipfsDir
+  }
+
+  getTotalRootStores () {
```

Comment:
I don't see how this code could possibly get the number of rootStores? It shouldn't be related to the data that is in the ipfs dir.

Reply:
Bad wording, it's the orbitDb directory, and it creates one directory per rootstore exactly. We can also get this information from the address server but since it's not timestamped at the db it's hard to see the number growing over time.